### PR TITLE
Added response models

### PR DESCRIPTION
**Pull request by ChrisMarsh82**
Added response models for endpoints (not forecasts)
Current issues:
- [ ] annual_rainfall_summaries - if dates aren't present they come back as {} not "" see #36 
- [x] crop_success_probabilities - prop_success always 0
- [ ] monthly_temperature_summaries - error when bringing back all summaries "unexpected value; permitted: 'mean_tmin', 'mean_tmax'"
- [x] season_start_probabilities - data not coming back in rows
- [ ]  season_start_probabilities - probability not going past .4 for test_1
- [ ]  extremes_summaries - Error in (function (daily, data_names, definitions, summaries)
- [x] station data - definitions to come back in the same format as metadata

### CODE REVIEW

**Comment by ChrisMarsh82** [comment]
@chrismclarke 

There are still the following issues to sort out on this. However, I think it should be merged as it works better than previous versions.

Here are the issues in epicsawrap https://github.com/IDEMSInternational/epicsawrap/issues/101

File: app/api/v1/endpoints/crop_success_probabilities/schema.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 class CropSuccessProbabilitiesParameters(BaseModel):
     country: country_code = "zm"
-    station_id: str = "16"
-    water_requirements: list[int] | None = None
-    planting_length: list[int] | None = None
-    planting_dates: list[int] | None = None
-    start_before_season: bool | None = None
+    station_id: str = "1"
```
Comment (chrismclarke) [inline app/api/v1/endpoints/crop_success_probabilities/schema.py:8]: nit(non-blocking) should this be `"test_1"` like the others?

File: app/api/v1/endpoints/extremes_summaries/router.py [hunk 2]
```diff
@@ -11,7 +12,7 @@
 router = APIRouter()
 
 
-@router.post("/")
+@router.post("/")#,response_model=ExtremesSummariesResponce)
```
Comment (chrismclarke) [inline app/api/v1/endpoints/extremes_summaries/router.py:15]: nit(blocking) - I think the `#` might be a typo

**Review by chrismclarke — APPROVED** [review]
Thanks @ChrisMarsh82 - all the new response models will definitely be a big help for getting integrated into the app.  I've updated locally and tested all the endpoints, they're all returning data so that should unblock dashboard development which is super handy. I haven't taken a close look at any of the data, but I'll try to start doing that more as I'm working on the picsa dashboard.

See a couple non-blocking comments inline, but otherwise I'd say good to merge and I'll add any feedback issues as they come up.

**Pull request merged**